### ISSUE TITLE
[bayesian-prior] Several TODOs (Trac #1126)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1120]
These should get done, made into tickets, or removed if not relevant anymore.  Have some of these actually been done, but the docs just not updated?

TODO
----
* Refit the zenith distributions for downgoing muons.
* Bayesian priors could be energy dependent.
* Priors could be used to force the solution to be contained or to be within a finite range from the detector or COG of the hits.
* Priors could be used to force the solution to be contained or to
* Figure how to use Bayesian priors in UHE analyses where you might want to use also events above the horizon.

Migrated from https://code.icecube.wisc.edu/ticket/1126
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "These should get done, made into tickets, or removed if not relevant anymore.  Have some of these actually been done, but the docs just not updated?\n\nTODO\n----\n* Refit the zenith distributions for downgoing muons.\n* Bayesian priors could be energy dependent.\n* Priors could be used to force the solution to be contained or to be within a finite range from the detector or COG of the hits.\n* Priors could be used to force the solution to be contained or to\n* Figure how to use Bayesian priors in UHE analyses where you might want to use also events above the horizon.\n", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[bayesian-prior] Several TODOs", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T17:03:16", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T16:31:26: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "claudio.kopper" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151014T12:55:50: kjmeagher changed owner from "olivas" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20151029T14:15:25: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 138934:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138934"
> Remove Todo list which were never going to get done, they all seemed unnescessary. clean up rst markup.
> 
> Fixes #1120, half of #1123
> ```


* kjmeagher changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
